package.json

```diff
@@ -107,10 +108,11 @@
     "ms": "0.7.1",
     "name-all-modules-plugin": "1.0.1",
     "node-sass": "4.5.3",
-    "notifications-panel": "1.2.7",
+    "notifications-panel": "2.1.0",
```

Comment:
Should this be `1.3.0` per https://github.com/Automattic/wp-calypso/pull/16870#issuecomment-341583193 by @gwwar?

> if the React upgrade lands first, target 2.1.0 for notifications-panel, otherwise 1.3.0

We're still on `react@15.6.2`

Reply:
i'm pretty sure all of the changes between 1.x and 2.x were backwards compatible: https://github.com/Automattic/notifications-panel/pull/191

Reply:
It makes more sense to stay on the 1.3.x line until we upgrade to React 16. I think 1.3.1 might be published soon. cc @kwight

Reply:
@gwwar ah yea. you are right. If it had said `react: ^15.0.0 || ^16.0.0` then it would be a different story but it just 16